hackathon_site/event/static/event/js/landing.js

```diff
@@ -71,7 +71,27 @@ $(document).ready(function () {
 function setCounter(countDownDate) {
     const now = new Date();
     const distance = countDownDate - now;
-    const days = Math.ceil(distance / (1000 * 60 * 60 * 24));
+    const days = Math.floor(distance / (1000 * 60 * 60 * 24));
+    const hours = Math.floor(distance / (1000 * 60 * 60));
+
+    if (hours < 1) {
+        const minutes = Math.floor(distance / (1000 * 60));
+        // Change to show minutes on the website
+        $("#day1").parent().remove();
+        $("#day2").html(Math.floor(minutes / 10));
+        $("#day3").html(minutes % 10);
+        $("#countdownUnit").html(minutes == 1 ? "Minute" : "Minutes");
+        return;
+    }
+
+    if (days < 2) {
```

Comment:
Should this be `days < 1`? Otherwise you'll start showing this at 47 hours remaining rather than 23 (unless that was your intent)